src/Our.Umbraco.HttpsRedirect/Events/ApplicationEventsHandler.cs

```diff
@@ -12,11 +12,22 @@ namespace Our.Umbraco.HttpsRedirect.Events
 {
     public class EventsHandler : ApplicationEventHandler
     {
-        public EventsHandler()
+
+        /// <summary>
+        /// Register event handler on start.
+        /// </summary>
+        /// <param name="httpApplicationBase">Umbraco application.</param>
+        /// <param name="applicationContext">Application context.</param>
+        protected override void ApplicationStarted(UmbracoApplicationBase umbracoApplication, ApplicationContext applicationContext)

```

Comment:
As far as I can tell, this is the only code change required to fix the original issue.  We can find out when the `ApplicationStarted` override was introduced (in Umbraco) and compile against that version.